easybuild/easyblocks/i/intel_compilers.py

```diff
@@ -120,12 +121,20 @@ def make_module_req_guess(self):
             os.path.join('compiler', 'lib', 'intel64_lin'),
         ]
         libdirs = [os.path.join(self.compilers_subdir, x) for x in libdirs]
+        libdirs.append(os.path.join(self.tbb_subdir, 'lib', 'intel64', 'gcc4.8'))
```

Comment:
This should use globbing since gcc4.8 may be changed.

Reply:
@bartoldeman +1 on this

Reply:
Used the get_tbb_gccprefix function for this now, like in the icc easyblock.